openshift-client/src/test/java/io/fabric8/openshift/client/Handlersv4Test.java

```diff
@@ -0,0 +1,81 @@
+/**
+ * Copyright (C) 2015 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.fabric8.openshift.client;
+
+import io.fabric8.kubernetes.api.model.v4_0.HasMetadata;
+import io.fabric8.kubernetes.clnt.v4_0.ResourceHandler;
+import io.fabric8.openshift.api.model.v4_0.Build;
+import io.fabric8.openshift.api.model.v4_0.BuildConfig;
+import io.fabric8.openshift.api.model.v4_0.DeploymentConfig;
+import io.fabric8.openshift.api.model.v4_0.Group;
+import io.fabric8.openshift.api.model.v4_0.ImageStream;
+import io.fabric8.openshift.api.model.v4_0.ImageStreamTag;
+import io.fabric8.openshift.api.model.v4_0.OAuthAccessToken;
+import io.fabric8.openshift.api.model.v4_0.OAuthAuthorizeToken;
+import io.fabric8.openshift.api.model.v4_0.OAuthClient;
+import io.fabric8.openshift.api.model.v4_0.PolicyBinding;
+import io.fabric8.openshift.api.model.v4_0.Project;
+import io.fabric8.openshift.api.model.v4_0.RoleBinding;
```

Comment:
Are you sure about this? I don't see any changes done in model with respect to this change. Could you please elaborate intention of having separate model for 4.0 openshift?